searx/templates/__common__/about.html

```diff
@@ -22,15 +22,14 @@ <h2>Technical details - How does it work?</h2>
 
 <h2>How can I make it my own?</h2>
 
-<p>Searx appreciates your concern regarding logs, so take the <a href="{{ brand.GIT_URL }}">code</a> and run it yourself! <br />Add your Searx to this <a href="{{ brand.SEARX_URL }}">list</a> to help other people reclaim their privacy and make the Internet freer!
+<p>Searx appreciates your concern regarding logs, so take the <a href="https://github.com/asciimoo/searx">code</a> and run it yourself! <br />Add your Searx to this <a href="{{ brand.PUBLIC_INSTANCES }}">list</a> to help other people reclaim their privacy and make the Internet freer!
```

Comment:
Some fork cache result using redis:
https://gitlab.e.foundation/e/cloud/my-spot/-/blob/master/searx/poolrequests.py#L166

A link to the original source code can be misleading.

Reply:
Good point / thanks!